src/models/UserModel.js

```diff
@@ -7,21 +7,22 @@ export const createUser = async (firstname, lastname, email, password) => {
 	const values = [firstname, lastname, email, password];
 
 	try {
-		const result = await dbConnection.query(query, values);
+		const result = await pool.query(query, values);
 		return result.rows[0];
 	} catch (error) {
-		logger.error(error);
+		return res.status(409).json({ error: "The email is already taken" });
```

Comment:
you should have anything to do with a `response` in the model. The model simply interacts with the database and return the data or result of the query.